README.md

```diff
@@ -84,7 +85,10 @@ chart-dirs:
 build-id: pr-42
 ```
 
-`ct install --config config.yaml`
+Config usage:
+
+    ct install --config config.yaml
+
```

Comment:
We should probably add that, if no config file is specified, `ct.yaml` (or any of the supported formats) is loaded from the current directory, `$HOME/.ct`, or `/etc/ct`, in that order, if found.